lib/stellar_core_commander/process.rb

```diff
@@ -184,8 +190,8 @@ def transaction_result(hex_hash)
     end
 
     Contract None => String
-    def http_host
-      "127.0.0.1"
+    def hostname
```

Comment:
Went with a new name here since it's not strictly the http host but rather the hostname in general since it's used for peering information as well.